### PR TITLE
Connection updates

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by stbenjam** [comment]
Isolated Client's Puppet should be 8140 to the puppet master

File: katello.dot [hunk 1]
```diff
@@ -27,27 +27,27 @@ node "server" {
...
     [Foreman SCAP] as foreman_scap
 
-    foreman_tasks -down-> dynflow
-    katello -> foreman
-    katello -> foreman_tasks
-    katello -> foreman_docker
+    foreman_tasks -down..> dynflow
+    katello ..> foreman
+    katello ..> foreman_tasks
+    katello ..> foreman_docker
```
Comment (jlsherrill) [inline katello.dot:33]: it seems sort of odd to list foreman_docker here, in this manner.  Is there any actual communication?

Comment (jlsherrill) [inline katello.dot:33]: for the plugins, you might use some non-directional arrow to indicate its 'connection' to foreman, as its not actually communication

File: katello.dot [hunk 2]
```diff
@@ -27,27 +27,27 @@ node "server" {
...
 
   server_apache ..> foreman
 
-  foreman -down..> postgres : localhost
-  candlepin -down..> postgres: localhost
+  foreman -[#red]down-> postgres : localhost
+  candlepin -[#red]down-> postgres: localhost
 
-  katello -down..> candlepin : localhost:8443
+  katello -[#red]down-> candlepin : localhost:8443
```
Comment (jlsherrill) [inline katello.dot]: This shouldn't be red should it?  looks like its using https to me.  I would think it would be cyan.

File: katello.dot (hunk 1 above)
Comment (ehelms) [inline katello.dot:33]: I was trying to leave `--->` as dependency requirements for all the various plugins. There are other arrow shapes provided by plantuml that I could try and use as well if you think one of those would be better at indicating this: http://plantuml.com/sequence.html#Arrow

Comment (jlsherrill) [inline katello.dot:33]: I think that is okay, i might would get rid of the direction though?

Comment (ehelms) [inline katello.dot:33]: If I do that, then how do I know who is requiring who?

File: katello.dot (hunk 2 above)
Comment (ehelms) [inline katello.dot]: Good catch, the same should be true of the katello - pulp connection (currently its set to one-way) ?

File: katello.dot (hunk 1 above)
Comment (jlsherrill) [inline katello.dot:33]: Ah, didn't realize we were trying to imply a requirement.  I guess its okay as is.

**Pull request merged**